Automattic/wp-calypso
# Assembler: Replace the default theme with the Creatio

Related to #

## Proposed Changes

* We want to replace the default theme of the Assembler with the “Creatio”

## Testing Instructions



* Go to Theme Showcase
* Scroll down and select Assembler CTA
* On the Assembler screen, select layout and styles
* Ensure everything works well
* Continue to the Site Editor
* Ensure your homepage is the same as the preview

## Pre-merge Checklist
